velox/expression/tests/ExprTest.cpp

```diff
@@ -1395,7 +1314,8 @@ TEST_F(ExprTest, cseEncodings) {
 
   auto indices = makeIndices({0, 0, 0, 1, 1});
   auto b = wrapInDictionary(indices, 5, makeFlatVector<int32_t>({11, 15}));
-  auto c = wrapInDictionary(indices, 5, makeFlatVector({"apple", "banana"}));
+  auto c = wrapInDictionary(
+      indices, 5, makeFlatVector<std::string>({"apple", "banana"}));
```

Comment:
Why are we changing this? Can the type be inferred automatically?

Reply:
It didn't compile without explicit template parameter.